### PR TITLE
Reviving es6-plato to support ES2020 features

### PR DESCRIPTION
**Pull request by pete-the-pete**
Addressing https://github.com/the-simian/es6-plato/issues/108

This seems like a lot, but much of it is formatting and small changes. I squashed a few of my commits but left some of the history; at one point I used imports and then converted back to commonJS.

The babel stuff is a little weird, and probably very specific to what I was doing at the time which was to use this on project with an estlintrc file that I couldn't/didn't want to modify at the time.

I'll continue to tweak this as necessary, but its at least capable of running on itself and the project i've been using it for.

### CODE REVIEW

**Comment by the-simian** [comment]
Quick question, why did you bother to change tabs to spaces in this PR? that seems out of scope for what this is about

**Comment by pete-the-pete** [comment]
Ya...I don't know, it was something I did thinking this would only ever be
an isolated fork.

**Comment by the-simian** [comment]
thanks for the reply, that's fair. I'll check it out pete, thank you so much for the PR

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "args": ["-d", "report", "-r", "lib/*"],
+            "program": "${workspaceFolder}/bin/plato"
+        }
+    ]
+}
```
Comment (jbrizio) [inline .vscode/launch.json:18]: I guess that not is really necessary to add

File: lib/assets/scripts/plato-file.js [hunk 2]
```diff
@@ -150,7 +150,7 @@ $(function() {
 	function addLintMessages(report) {
 		var lines = {};
 
-		console.log(report);
+		// console.log(report);
```
Comment (jbrizio) [inline lib/assets/scripts/plato-file.js:153]: Not is a good practice maintain code commented

**Pull request closed without merging**